src/views/hierachicalPackageRootSubNode.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { Jdtls } from "../java/jdtls";
+import { INodeData, NodeKind } from "../java/nodeData";
+import { PackageTreeNode } from "../java/packageTreeNode";
+import { DataNode } from "./dataNode";
+import { ExplorerNode } from "./explorerNode";
+import { FileNode } from "./fileNode";
+import { FolderNode } from "./folderNode";
+import { ProjectNode } from "./projectNode";
+import { TypeRootNode } from "./typeRootNode";
+
+export class HierachicalPackageRootSubNode extends DataNode {
+
+    public packageTree: PackageTreeNode;
```

Comment:
This class code is duplicate with the HierarchicalPackageRootNode, could we model these two types in one class?